Library/Homebrew/extend/ENV/std.rb

```diff
@@ -144,13 +138,6 @@ def no_optimization
   end
   alias generic_no_optimization no_optimization
 
-  def libxml2
-  end
-
-  def x11
-  end
-  alias libpng x11
-
   # we've seen some packages fail to build when warnings are disabled!
   def enable_warnings
```

Comment:
Technically a valid API in `env :std`. I guess there's an argument that we should remove any of these that aren't still used/needed in homebrew/core (e.g. `ENV.libxml2` is because we use in in `test do` which uses `env :std`). Thoughts?